.github/workflows/haros.yml

```diff
@@ -0,0 +1,58 @@
+name: Haros Analysis
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  haros:
+    runs-on: ubuntu-20.04
+    container:
+      image: swarmus/embedded
+    steps:
+      - name: Create catkin_ws
+        run: |
+          mkdir -p catkin_ws/src
+          cd catkin_ws
+          . /opt/ros/$ROS_DISTRO/setup.sh
+          catkin_make
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          path: 'catkin_ws/src/SwarmUS-ROS'
+
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+        working-directory: 'catkin_ws/src/SwarmUS-ROS'
+
+      - name: Install Haros
+        run: |
+          . /opt/ros/$ROS_DISTRO/setup.sh
+          apt-get update && apt-get install -y python2 libmagic-dev
+          wget https://bootstrap.pypa.io/get-pip.py
```

Comment:
Why are we not using pip from the ubuntu repository?

Reply:
Getting the right package to install pip/pip2 with python3/python2 has always been a little messy for me. Sometimes the `pip` command refers to `pip3` really, or `pip2`, and so on. This installer just makes sure that there is no conflict between pip2 and pip3.

Reply:
Ok and I just checked an pip isn't even included on LST 20.04.  So that clears it up!